src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -462,17 +436,21 @@ bool EmitAnchors()
                 {
```

Comment:
should any of these switch statements end with
```c#
default:
     Debug.Fail("missing case");
```
?

Reply:
These only handle a subset of cases, so I can't add such a debug fail here.  I did add them at the higher level where I could.